Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -266,23 +267,6 @@ - (BOOL)hasQuotedAttachment
     return self.viewItem.hasQuotedAttachment;
 }
 
-- (BOOL)hasQuotedAttachmentThumbnail
```

Comment:
This class is really bloated, so it's nice to pull out the quoted reply logic into its own class.